.gitlab-ci.yml

```diff
@@ -61,3 +61,10 @@ production:
         K8S_SECRET_DEFAULT_FROM_EMAIL: "no-reply@hel.ninja"
         K8S_SECRET_FIELD_ENCRYPTION_KEYS: "$GL_STABLE_FIELD_ENCRYPTION_KEYS"
         K8S_SECRET_FORCE_SCRIPT_NAME: "/profiili"
+        K8S_SECRET_MEDIA_URL: "/profiili/media/"
+        K8S_SECRET_STATIC_URL: "/profiili/static/"
+        K8S_SECRET_CSRF_COOKIE_NAME: "profiili-prod-csrftoken"
+        K8S_SECRET_CSRF_COOKIE_PATH: "/profiili/"
+        K8S_SECRET_CSRF_COOKIE_SECURE: True
+        K8S_SECRET_SESSION_COOKIE_NAME: "profiili-prod-sessionid"
+        K8S_SECRET_SESSION_COOKIE_PATH: "/profiili/"
```

Comment:
Secure session cookie is a good idea also.

Reply:
I changed the secure session cookie setting so that it can now be set in the environment variables. I also changed the Gitlab config so that test and prod environments are using secure cookies.